index.js

```diff
@@ -1181,6 +1191,73 @@ SqlPort.prototype.doc = function(schema) {
 };
 
 SqlPort.prototype.tryConnect = function() {
+    if (this.config.db) {
+        this.config.db.beforeConnect = c => {
```

Comment:
where this is used ?

Reply:
ut-mssql is calling this before connecting to DB